src/test/java/runners/LoginTestRunner.java

```diff
@@ -0,0 +1,8 @@
+package runners;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(features="src/test/resources/Features/LoginTest.feature",glue={"StepDefinition"})
+public class LoginTestRunner extends AbstractTestNGCucumberTests {
```

Comment:
Ditto. I guess this Runner can be removed as you have all setup in CucumberTestRunner.